plugins/examples/headlamp-events/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "headlamp-events",
```

Comment:
Maybe this could be `@headlamp-k8s/headlamp-events`

Reply:
Maybe we should have a prefix for all example plugins, e.g. `@headlamp-k8s/plugin-examples.headlamp-events`?

Reply:
Hrmm. Yeah, so we can see that it is a plugin example in the name. As apposed to something completely different (like a lint config package.

I like how `cd plugins/examples` is kind of in the name too.

Another option could be: `@headlamp-k8s/headlamp-events-plugin-example`.
But... probably we should have this naming discussion outside of this PR.

Reply:
Right. We'd need to rename all example plugins anyway, and this is only relevant if we decide to publish them. So let's keep a pin on the conversation and get back to it when we decide to publish the packages.